packages/components/src/base-control/index.js

```diff
@@ -3,11 +3,11 @@
  */
 import classnames from 'classnames';
 
-function BaseControl( { id, label, help, className, children } ) {
+function BaseControl( { id, label, hideLabelFromVision, help, className, children } ) {
 	return (
 		<div className={ classnames( 'components-base-control', className ) }>
 			<div className="components-base-control__field">
-				{ label && id && <label className="components-base-control__label" htmlFor={ id }>{ label }</label> }
+				{ label && id && <label className={ classnames( 'components-base-control__label', hideLabelFromVision && 'screen-reader-text' ) } htmlFor={ id }>{ label }</label> }
```

Comment:
This line is getting quite busy, it would be great to break it up onto multiple lines. The call to classnames could also be written like this, which is a pattern used more commonly across the codebase:
```jsx
{ classnames( 'components-base-control__label', {
    'screen-reader-text': hideLabelFromVision,
} ) }
```